hadoop-tools/hadoop-distcp/src/site/markdown/DistCp.md.vm

```diff
@@ -440,6 +440,23 @@ $H3 Copy-listing Generator
   of DistCp differs here from the legacy DistCp, in how paths are considered
   for copy.
 
+  One may also customize the filtering of files which shouldn't be copied
+  by passing the current supported implementation of CopyFilter interface
+  or a new implementation can be written. This can be specified by setting the
+  `distcp.filters.class` in the DistCpOptions:
+
+  1. `distcp.filters.class` to RegexCopyFilter. If you are using this implementation,
+     you will have to pass along CopyFilter`distcp.filters.file` which contains the regex
```

Comment:
1. add space before the tick
2. use `quotes` around all classnames

Reply:
Done